scene/2d/audio_stream_player_2d.cpp

```diff
@@ -488,6 +534,12 @@ void AudioStreamPlayer2D::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("get_stream_playback"), &AudioStreamPlayer2D::get_stream_playback);
 
+	ClassDB::bind_method(D_METHOD("set_distance_drawing_enabled", "distance_drawing_enabled"), &AudioStreamPlayer2D::set_distance_drawing_enabled);
+	ClassDB::bind_method(D_METHOD("is_distance_drawing_enabled"), &AudioStreamPlayer2D::is_distance_drawing_enabled);
+
+	ClassDB::bind_method(D_METHOD("set_attenuation_drawing_enabled", "attenuation_drawing_enabled"), &AudioStreamPlayer2D::set_attenuation_drawing_enabled);
+	ClassDB::bind_method(D_METHOD("is_attenuation_drawing_enabled"), &AudioStreamPlayer2D::is_attenuation_drawing_enabled);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("set_distance_drawing_enabled", "enabled"), &AudioStreamPlayer2D::set_distance_drawing_enabled);
	ClassDB::bind_method(D_METHOD("is_distance_drawing_enabled"), &AudioStreamPlayer2D::is_distance_drawing_enabled);

	ClassDB::bind_method(D_METHOD("set_attenuation_drawing_enabled", "enabled"), &AudioStreamPlayer2D::set_attenuation_drawing_enabled);
	ClassDB::bind_method(D_METHOD("is_attenuation_drawing_enabled"), &AudioStreamPlayer2D::is_attenuation_drawing_enabled);
```
Less redundant and that's how it's done usually.